From 839e900510a47056f885ff86cc31cb7096862f53 Mon Sep 17 00:00:00 2001
From: Ghanshyam Mann <gmann@ghanshyammann.com>
Date: Fri, 15 Sep 2023 10:23:01 -0700
Subject: [PATCH] Fix the unstable_test syntax for
 test_live_migration_with_trunk

test_live_migration_with_trunk test was marked
as unstable in
- https://review.opendev.org/c/openstack/tempest/+/895167

but it used the wrong syntax for unstable_test
decorator.

Change-Id: I8f4f3cafe8265984470bf45a9d85b3c9f765a251
---
 tempest/api/compute/admin/test_live_migration.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tempest/api/compute/admin/test_live_migration.py b/tempest/api/compute/admin/test_live_migration.py
index 2826f56bc..96990bf87 100644
--- a/tempest/api/compute/admin/test_live_migration.py
+++ b/tempest/api/compute/admin/test_live_migration.py
@@ -253,6 +253,9 @@ class LiveMigrationTest(LiveMigrationTestBase):
         port = self.ports_client.show_port(port_id)['port']
         return port['status'] == 'ACTIVE'
 
+    @decorators.unstable_test(bug='2024160')
+    @decorators.unstable_test(bug='2033887')
+    @decorators.attr(type='multinode')
     @decorators.idempotent_id('0022c12e-a482-42b0-be2d-396b5f0cffe3')
     @utils.requires_ext(service='network', extension='trunk')
     @utils.services('network')
-- 
2.51.0

